server/src/errorUtils.ts

```diff
@@ -14,7 +14,9 @@ export interface FormatErrorMetadata {
 export function handleError(connection: LS.Connection, value: unknown, action: string): void {
     let userMessage: string;
 
-    if (value instanceof Error) {
+    if (PQP.CommonError.isCommonError(value) && value.innerError instanceof PQP.CommonError.CancellationError) {
+        return;
```

Comment:
It's not worth noting these in our traces?

Reply:
Updated to include additional tracing.